packages/os/autoscaling-warm-pool.service

```
@@ -0,0 +1,17 @@
+[Unit]
+Description=Run tasks reliant on IMDS
+After=network-online.target settings-applier.service
+Requires=settings-applier.service
+Wanted=network-online.target
+Before=kubelet.service
+# We only want to run once, at first boot. This file is created by shibaken
+# after a successful run.
+ConditionPathExists=!/var/lib/bottlerocket/shibaken.ran
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/shibaken boot-shibaken
+RemainAfterExit=true
+
+[Install]
+WantedBy=multi-user.target
```

```suggestion
WantedBy=multi-user.target

```
A couple files are missing the end-of-file newline.

---

To be clear, we just want one newline at the end of the file, not two newlines (which is what shows in @jpculp's suggestion.) This may be a configuration option in your text editor that needs to be set.